sunkit_pyvista/plotter.py

```diff
@@ -140,3 +144,38 @@ def plot_solar_axis(self, length=2.5, arrow_kwargs={}, **kwargs):
                          scale='auto',
                          **defaults)
         self.plotter.add_mesh(arrow, **kwargs)
+
+    def plot_field_lines(self, gong_map, lon, lat, radius, nrho=25, rss=2.5, **kwargs):
+        """
+        Plots the field lines from `pfsspy`.
+
+        Parameters
+        ----------
+        gong_map : `pfsspy.map.GongSynopticMap`
+            to produce the field lines
+        lon : `astropy.units.quantity`
+        lat : `astropy.units.quantity`
+        radius : `float`
+        nrho : `int`
+        rss : `int`
+        **kwargs :
+            Keyword arguments are handed to `pyvista.Plotter.add_mesh`.
+        """
+        tracer = tracing.PythonTracer()
+        input_ = pfsspy.Input(gong_map, nrho, rss)
+        output_ = pfsspy.pfss(input_)
+
+        seeds = SkyCoord(lon, lat, radius*R_sun,
+                         frame=gong_map.coordinate_frame)
+
+        field_lines = tracer.trace(seeds, output_)
+        grid_data = []
+
+        for field_line in field_lines:
+            grid = self._coords_to_xyz(field_line.coords.ravel())
+            grid_data.append(grid)
+
+        grid_data = np.concatenate(grid_data)
+        field_line_mesh = pv.StructuredGrid(grid_data[:, 0], grid_data[:, 1], grid_data[:, 2])
+
+        self.plotter.add_mesh(field_line_mesh, kwargs)
```

Comment:
I think you are going to want to add each field line as a separate mesh, because otherwise you will get lines drawn between the end of one field line and the start of the next. Perhaps I'm wrong though?

Reply:
Ah yeah, I made this change locally, I forgot to commit sorry